.github/workflows/gradle.yml

```diff
@@ -108,13 +108,13 @@ jobs:
 
     - name: VIVIDUS tasks validation
       run: |
-         ./gradlew validateKnownIssues printSteps countSteps -x testVividusInitialization -Pvividus.bdd.variables.global.iterationLimit=3 -Pvividus.bdd.variables.global.target-platform=ios -p vividus-tests
```

Comment:
is there any issue about the behavior with -P?

Reply:
Powershell parses input args and splits them at dot `.` for some reason:
```log
Run ./gradlew validateKnownIssues printSteps countSteps -x testVividusInitialization -Pvividus.bdd.variables.global.iterationLimit=3 -Pvividus.bdd.variables.global.target-platform=ios -p vividus-tests
  ./gradlew validateKnownIssues printSteps countSteps -x testVividusInitialization -Pvividus.bdd.variables.global.iterationLimit=3 -Pvividus.bdd.variables.global.target-platform=ios -p vividus-tests
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    JAVA_HOME: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\16.0.2-7\x64
Starting a Gradle Daemon (subsequent builds will be faster)


FAILURE: Build failed with an exception.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.


* What went wrong:
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
Task '.bdd.variables.global.iterationLimit=3' not found in project ':vividus-tests'.


See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings
* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 21s
```